### ISSUE TITLE
Any comment about upcoming Git Rev News edition 101

### ISSUE THREAD

**Issue by chriscool** [git/git.github.io#652]
A currently mostly empty draft is there:

https://github.com/git/git.github.io/blob/master/rev_news/drafts/edition-101.md

Feel free to comment in this issue, suggest topics, suggest persons to interview, or use the edit button (that looks like a pen) to edit and create a pull request with the changes you would like.

Let's try to publish this edition on Wednesday July 26th 2023!

Thanks!

cc @jnareb @mjaix @sivaraam @gitster

**Comment by jnareb** [comment]
Topics we might want to cover (but might turn out to be not finished at publication date):

- [ ] [Introduce Git Standard Library](https://public-inbox.org/git/7c8d6012-5bb5-9ee5-eda1-80e46c4b6981@jeffhostetler.com/t/#u)

**Comment by chriscool** [comment]
@jnareb thanks for the suggestion! I will take a look.

**Comment by chriscool** [comment]
Draft sent:

https://lore.kernel.org/git/CAP8UFD1M3sRPEJ42v2=pQ_CQYY+PN64Swx70O7D_=VfbhSwKLg@mail.gmail.com/

**Comment by chriscool** [comment]
@jnareb the article I wrote is indeed about the thread you suggested. Thanks again!

**Comment by jnareb** [comment]
I wonder if it be worth including the series of instructions on how to obfuscate timestamps in Git called [Git-Privacy](https://git.nicholasjohnson.ch/git-privacy/tree/README.md) - see also [discussion on git mailing list](https://public-inbox.org/git/CTZ9RD9RQ5UO.3OIJX50PKMIR0@anonymous/t/).

**Comment by jnareb** [comment]
We should mention [Git Virtual Contributor's Summit 2023](https://public-inbox.org/git/ZMATKIaU1A1D0wJg@nand.local/#r) which is this year (?) replacing in-person Git Merge, hosted by GitHub every year since at least 2015 until 2022.

**Comment by jnareb** [comment]
My links have landed in  02db9c0 (and one release announcement for gitolite).

**Comment by chriscool** [comment]
@jnareb thanks for your links!

We have an email reply to the draft email I sent by Kristoffer Haugsbakk about one of the links:

https://lore.kernel.org/git/a5c359d6-6c21-489c-a4a3-a21219ab727a@app.fastmail.com/

Should we keep the link discussed in that email?

**Comment by jnareb** [comment]
> We have an email reply to the draft email I sent by Kristoffer Haugsbakk about one of the links:
> 
> https://lore.kernel.org/git/a5c359d6-6c21-489c-a4a3-a21219ab727a@app.fastmail.com/
> 
> Should we keep the link discussed in that email?

I agree that the article does not provide enough reason on why one would want an empty commit, and is so short that it probably better be removed.

Done in 5df08c3

**Comment by chriscool** [comment]
> > We should mention [Git Virtual Contributor's Summit 2023](https://public-inbox.org/git/ZMATKIaU1A1D0wJg@nand.local/#r) which is this year (?) replacing in-person Git Merge, hosted by GitHub every year since at least 2015 until 2022.
> 
> Yeah, I will take care of mentioning it. (Also not sure there was a Git Merge in 2021 due to Covid. BTW I think there was also one in Berlin in 2013.)

Done in: 1b99c75466b9a97c2eaf8050fe903233366155c6

**Comment by chriscool** [comment]
> I wonder if it be worth including the series of instructions on how to obfuscate timestamps in Git called [Git-Privacy](https://git.nicholasjohnson.ch/git-privacy/tree/README.md) - see also [discussion on git mailing list](https://public-inbox.org/git/CTZ9RD9RQ5UO.3OIJX50PKMIR0@anonymous/t/).

@jnareb I had no time to look at this, but I copied your comment on the issue for the next edition. Thanks!

**Comment by chriscool** [comment]
Published and announced in: https://public-inbox.org/git/CAP8UFD19S7i9iu-m_sRsCb_c-mbChWvk23vdWLX+iuUarH5RpA@mail.gmail.com/
